ravendb/resource_ravendb_server.go

```diff
@@ -229,23 +234,32 @@ func resourceServerDelete(ctx context.Context, d *schema.ResourceData, meta inte
 	return sc.RemoveRavenDbInstances()
 }
 
-func convertNode(node NodeState) map[string]interface{} {
+func convertNode(node NodeState, index int) map[string]interface{} {
 	return map[string]interface{}{
-		"host":        node.Host,
-		"license":     base64.StdEncoding.EncodeToString(node.Licence),
-		"settings":    node.Settings,
-		"certificate": base64.StdEncoding.EncodeToString(node.ClusterCertificate),
-		"http_url":    node.HttpUrl,
-		"tcp_url":     node.TcpUrl,
-		"assets":      node.Assets,
-		"unsecured":   node.Unsecured,
-		"version":     node.Version,
-		"failed":      node.Failed,
+		"host":               node.Host,
+		"license":            base64.StdEncoding.EncodeToString(node.Licence),
+		"settings":           node.Settings,
+		"certificate_holder": node.ClusterSetupZip[string(index+'A')].String(),
```

Comment:
Can we add some spacing here?
```suggestion
		"certificate_holder": node.ClusterSetupZip[string(index + 'A')].String(),
```

Reply:
extracted it to a variable so that when formatting it won't go back to this form - `string(index+'A)`